frontend/style.js

```diff
@@ -1,22 +1,6 @@
 import { StyleSheet } from 'react-native';
 
 const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
```

Comment:
It might be better to leave at least some of these styles, especially ones like container, in this file. Just because I use them to quickly lay out components when I'm testing functionality like AWS integration for example, and I don't want to have to worry about styling my components and I just want them to appear on the screen somewhere where I can see and interact with. We can always remove legacy code or style later.